docker/main/partitionerservice/Dockerfile

```diff
@@ -8,18 +8,34 @@ ARG partitioner_service_package_name
 ARG partitioner_service_listener_port
 
 # Currently need Git installed to retrieve some package transitive dependencies
-RUN apk --no-cache --update-cache add git
+RUN apk update && apk --no-cache upgrade && apk --no-cache add git libstdc++ openblas-dev proj-dev gdal-dev
 
 WORKDIR /code
 
 # Copy custom built packages from previous stage FROM external sources image
 COPY --from=sources /DIST /DIST
 
 # Install custom and generally available packages, starting with any custom from external source image
-RUN pip install --no-index --find-links=/DIST ${partitioner_service_package_name} \
+RUN pip install --upgrade pip \
+    && pip install --no-cache --no-index --find-links=/DIST ${partitioner_service_package_name} \
     # After eventually installing all custom packages like this, clean up ... \
     && rm -r /DIST
 
 EXPOSE ${partitioner_service_listener_port}
 
-ENTRYPOINT [ "python3", "-m", "${partitioner_service_module_name}" ]
\ No newline at end of file
+RUN echo "#!/bin/sh" > /helper.sh \
```

Comment:
what is this helper for?

Reply:
I think I added it for debugging purposes at one point.  Probably should be cleaned up, but I'll leave it for now to avoid the entropy at this point.